app/views/properties/show.html.erb

```diff
@@ -11,6 +11,7 @@
   <div class='flex flex-row'>
     <div id='Map' class='flex flex-col align-self: center'>
       <p>Map</p>
```

Comment:
Interesting, the map needs to have lat and lon passed in the show? I may need some help figuring this out next time we are all together.